src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -489,7 +489,6 @@ private ContextMenu createContextMenuForGroup(GroupNodeViewModel group) {
         contextMenu.getItems().addAll(
                 factory.createMenuItem(StandardActions.GROUP_EDIT, new ContextAction(StandardActions.GROUP_EDIT, group)),
                 removeGroup,
-                factory.createMenuItem(StandardActions.GROUP_EDIT, new ContextAction(StandardActions.GROUP_EDIT, group)),
```

Comment:
Huh, this must have been an artifact from refactoring. Thanks!

Reply:
You're welcome